azure/resource-containers/pf-management-group/functions/private/update-pfazmanagementgroup.ps1

```diff
@@ -0,0 +1,15 @@
+Function Update-PfAzManagementGroup{
+    Param (
```

Comment:
This is the private function now that will use the az module to update the management group from Azure if persiststate is set to true